docs/guidelines/styles.rst

```diff
@@ -1,28 +1,133 @@
-Guidelines
-==========
-
-* Use **4 space indentation** and **not** tabs
-* Use **underscores** for scss file naming
-* Use **double-quotes** ``"`` for all text values
-* Use **dashes** to separate class/id names, **not** camelCase or underscore
-* Do **not** overuse nesting! If you got only one instance, use one line
-* Always add a space after the colon
-* Only write one css property per line
-* Keep ``sass/layout/`` clean and use the available structure
-* Use ``sass/sites/`` for theme based or specific styles
-* Define settings within ``sass/settings/``
-* Avoid referencing css using their parent like div.container
-* Use shorthands for values like ``#ccc`` or ``white``
-* Use full words instead of shorthands like ``number`` instead of ``nr``
-* Avoid using universal selectors for maintainability/performance reasons
-* Use progressive enhancement whenever possible
-* Validation is not required but nice
-
-
-Style
------
-
-Use block-style and group elements underneath:
+Styles
+======
+
+.. note::
+
+    In addition to the :doc:`general` guidelines, the following sections describe stylesheet specific rules.
+
+
+Naming
+------
+
+.. admonition:: Files
+    :class: `important`
+
+    - Use **underscores** for scss file naming
+    - Use only **dashes** to separate class/ID names
+
+Valid
+*****
+
+.. code-block:: text
+
+    search.scss, marketing_site.scss or theme_dark_blog.scss
+
+.. code-block:: html
+
+    class="blog blog-item blog-item-featured"
+
+Invalid
+*******
+
+.. code-block:: text
+
+    Search.scss, marketingSite.scss or theme-dark-blog.scss
+
+.. code-block:: html
+
+    class="blog blogItem blog_item__featured"
+
+
+Nesting
+-------
+
+.. admonition:: Tree
+    :class: `important`
+
+    - Don't overuse nesting, nest elements to a max of **4 indends**
```

Comment:
always feels like a bit too much
2-3 is ok, 4 is extreme, usually it means that you are overspecifying something (or battling bootstrap)

also typo